From 81cd08aa577b4b91827c822fd05378f0c0085ab3 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 3 Apr 2012 02:28:13 +0300
Subject: [PATCH 119/311] fix two of three red spots reported by ReviewBoard

The third one is within "XXX" comment that will be deleted anyway.
Let red colour emphasize XXX-ness of the comment.
---
 conf/confc.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/conf/confc.h b/conf/confc.h
index 630fa28..0f03703 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -111,14 +111,14 @@
  * struct c2_confc confc;
  *
  * startup(...) {
- * 	rc = c2_confc_init(&confc, "profile-name", "confd-endpoint", group);
- * 	...
+ *     rc = c2_confc_init(&confc, "profile-name", "confd-endpoint", group);
+ *     ...
  * }
  *
  * ... Access configuration objects using confc interfaces. ...
  *
  * shutdown(...) {
- * 	c2_confc_fini(confc);
+ *     c2_confc_fini(confc);
  * }
  * @endcode
  *
-- 
1.8.3.2

